plexapi/client.py

```diff
@@ -94,7 +94,7 @@ def connect(self, timeout=None):
             raise Unsupported('Cannot reload an object not built from a URL.')
         self._initpath = self.key
         data = self.query(self.key, timeout=timeout)
-        if not data:
+        if not len(data):
```

Comment:
Should this be `if data is not None` because `len(None)` will raise `TypeError`?

Reply:
Maybe `if data is None`? I assume this should match the error case.